db/schema.rb

```diff
@@ -155,7 +155,7 @@
     t.string   "service"
     t.string   "identifier"
     t.boolean  "admin",        default: false
-    t.string   "language",     default: "en"
+    t.string   "language"
```

Comment:
This one is making the tests fail, which doesn't mean the other ones shouldn't be fixed too.